html/changelogs/FurryDog-woof.yml

```diff
@@ -33,4 +33,6 @@ delete-after: True
 # Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - rscadd: "Added a log message that will appear for you and everyone who has you in their sight if you examine something"
```

Comment:
You might want to check this out. It apparently removed something?

Reply:
This is changelog .-.

Reply:
Pretty sure it's removing the old suits. Also, HAZMAT suits are ONLY a yellow. You won't find white or grey hazmat suits ever. Or blue, or anything. They're meant to be ambiguous, and are only colored in a military setting, to black. At least in most places.

Hazmat Suits are Class A suits, class B suits would be for chemical work, class C suits are for less volatile things where only a gas mask and a sterile suit is required (IE laboratories). Might be relevant to check that in the future :^)

Reply:
I don't think so.